scripts/exsrw_nexus_gfs_sfc.sh

```diff
@@ -8,7 +8,11 @@
 #-----------------------------------------------------------------------
 #
 . ${USHsrw}/source_util_funcs.sh
-source_config_for_task "cpl_aqm_parm|task_nexus_gfs_sfc" ${GLOBAL_VAR_DEFNS_FP}
+for sect in user nco platform workflow nco global verification cpl_aqm_parm \
+  constants fixed_files grid_params \
+  task_nexus_gfs_sfc ; do
```

Comment:
There is no `task_nexus_gfs_sfc` in `config_defaults.yaml`.

```suggestion
  constants fixed_files grid_params ; do
```